lib/matplotlib/__init__.pyi

```diff
@@ -96,7 +96,7 @@ def rc_file(
 @contextlib.contextmanager
 def rc_context(
     rc: dict[str, Any] | None = ..., fname: str | Path | os.PathLike | None = ...
-): ...
+) -> Generator[None, None, None]: ...
```

Comment:
Should we be doing `AbstractContextManager` here instead of the decorator?

Reply:
If I do that, then mypy complains that `AbstractContextManager[None]` is not callable when used as a decorator. I wonder if we should revert the others as well.